eventyay_business/forms.py

```
@@ -104,3 +104,37 @@ def clean(self):
+        fields = [
+            "organizer",
+            "tier_version",
+            "status",
+            "billing_interval",
+            "currency",
+            "starts_at",
+            "ends_at",
+            "cancel_at",
+            "stripe_customer_id",
+            "stripe_subscription_id",
+        ]
```

Ruff (0.16.3)

[warning] Mutable default value for class attribute

(RUF012)